src/gesture-recognizers.js

```diff
@@ -691,42 +693,44 @@ export class PinchRecognizer extends GestureRecognizer {
   /** @override */
   onTouchStart(e) {
     const touches = e.touches;
-    if (!touches || touches.length != 2) {
+    if (!touches || touches.length > 2) {
       return false;
+    } else if (touches.length == 1) {
+      return true;
+    } else {
+      this.startTime_ = Date.now();
+      this.startX1_ = touches[0].clientX;
+      this.startY1_ = touches[0].clientY;
+      this.startX2_ = touches[1].clientX;
+      this.startY2_ = touches[1].clientY;
+      return true;
     }
-    this.startTime_ = Date.now();
-    this.startX1_ = touches[0].clientX;
-    this.startY1_ = touches[0].clientY;
-    this.startX2_ = touches[1].clientX;
-    this.startY2_ = touches[1].clientY;
-    return true;
   }
 
   /** @override */
   onTouchMove(e) {
     const touches = e.touches;
-    if (!touches || touches.length != 2) {
-      return false;
```

Comment:
Instead of returning `false`, which causes `PinchRecognizer` to stop tracking this event series, we return `true`, but do not act on the touch event unless the number of touches is explicitly 2.